lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -5875,108 +6123,104 @@ ADContext::declareExternalAssociatedFunction(
   return assocFn;
 }
 
-static SILFunction* createJVP(
+static SILFunction *createEmptyVJP(
```

Comment:
The git diff here is acting funny in the next several lines, may be hard to follow but I more so made a copy of `createEmptyVJP` and named it `createEmptyJVP` and removed the previous `createJVP` to more closely resemble how `VJPEmitter` looks like and `JVPEmitter` will become.